client-sdk/ts-web/core/babel.config.js

```diff
@@ -0,0 +1 @@
+module.exports = require('../babel.config')
```

Comment:
Should it be done in this PR or in another one?

We could do this in `package.json`  - this would also lint and fix the playground : 

```json
    "scripts": {
        "lint": "prettier --check '**/src/**.ts' '*.js' '**/src/*.js'",
        "fmt": "prettier --write '**/src/**.ts' '*.js' '**/src/*.js'"
    },
```